syntaxes/http.tmLanguage

```diff
@@ -124,7 +124,7 @@
             </dict>
             <dict>
                 <key>match</key>
-                <string>^\s*#{1,}\s*(?:((@)name)\s+(\S+))?.*$</string>
```

Comment:
This line is used for the comment line starts with `#`, you should update both places in the `http.tmLanguage` file.